desktop-environment.el

```diff
@@ -152,6 +152,10 @@ The shell command should let the user interactively select the
 portion of the screen."
   :type 'string)
 
+(defcustom desktop-environment-screenshot-delay-argument " -d %d"
+  "Format string to be appended to the desktop environment shell command in order to delay the screenshot."
```

Comment:
can you please reference `desktop-environment-screenshot-command` and `desktop-environment-screenshot-partial-command` in the docstring? Similarly, please add a reference to `desktop-environment-screenshot-delay-argument` in the 2 other user options.e

Reply:
Should be resolved as of 9348495